logic/SituationManager.py

```diff
@@ -0,0 +1,47 @@
+import json
+import random
+
+from situation import Situation
+
+
+class SituationManager():
+    """SituationManager class"""
+
+    def __init__(self):
+        self.current_situation = None
+        self.situation_counter = 0
```

Comment:
situation_counter should be part of state in StateManager, we can pass current_situation to the get_situation method.